draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1626,12 +1633,17 @@ client uses `get-all-by-hash`, then it will already have the new STH.
 
 ### Evaluating compliance
 
-To be considered compliant, a certificate MUST be accompanied by at least one
-valid SCT. A certificate not accompanied by any valid SCTs MUST NOT be
-considered compliant by TLS clients.
+To be considered compliant with this specification, a certificate MUST be
+accompanied by evidence that it has been included in one or more logs. There
+are two types of proof that a client might require:
 
-A TLS client MUST NOT evaluate compliance if it did not send both the
-`transparency_info` and `status_request` TLS extensions in the ClientHello.
+1. A log inclusion proof to a Signed Tree Head
+2. A Signed Certificate Timestamp
+
+It is up to a client's local policy which forms of evidence are sufficient for
+compliance.  For example, a client might require certificates to come with some
+minimum number of SCTs from among a collection of trusted logs, or require a
+single inclusion proof to an audited tree head.
```

Comment:
"SCTs...or...inclusion proof"?  AIUI, to verify inclusion a client needs an inclusion proof, an STH, the certificate *and* the corresponding SCT.